AlgorithmFactory/Loader.cs

```diff
@@ -184,6 +185,10 @@ private bool TryCreatePythonAlgorithm(string assemblyPath, out IAlgorithm algori
             }
             catch (Exception e)
             {
+                // perform exception interpretation for error in module import
+                var interpreter = StackExceptionInterpreter.CreateFromAssemblies(AppDomain.CurrentDomain.GetAssemblies());
+                e = interpreter.Interpret(e, interpreter);
+
                 Log.Error(e);
                 errorMessage = $"Loader.TryCreatePythonAlgorithm(): Unable to import python module {assemblyPath}. {e.Message}";
```

Comment:
Perhaps instead of `e.Message` we should use `interpreter.GetMessageHeader(e)` -- this will produce to collated exception message containing the messages from inner exceptions as well.